<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WOrder</class>
 <widget class="QWidget" name="WOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="6">
       <widget class="C5LineEdit" name="leCash">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbPartner">
        <property name="text">
         <string>Partner</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Cash (F3)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Change</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Total</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>108</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2" colspan="5">
       <widget class="C5LineEdit" name="lePartner">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="C5LineEdit" name="leCard">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Card (F4)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="C5LineEdit" name="leTotal">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="C5LineEdit" name="leChange">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbCustomer">
        <property name="text">
         <string>Customer</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="3">
       <widget class="C5LineEdit" name="leCustomer">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLabel" name="lbDisc">
        <property name="text">
         <string>Discount</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="C5LineEdit" name="leDisc">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="C5TableWidget" name="tblGoods">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget::item:selected{
 background-color: #005e37;
color:white;
font-size:13pt;
};</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Goods code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Goods name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Qty</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Unit</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>TaxDept</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>AdgCode</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="wimage" native="true">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lbGoodsImage">
        <property name="text">
         <string>Image</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="btnInfo">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/res.qrc">
       <normaloff>:/info.png</normaloff>:/info.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEdit</class>
   <extends>QLineEdit</extends>
   <header>c5lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>C5TableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">c5tablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
